build.sbt

```diff
@@ -16,5 +16,6 @@ parallelExecution in Test := false
 
 libraryDependencies ++= Seq(
   "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-  "com.novocode" % "junit-interface" % "0.11" % "test"
+  "com.novocode" % "junit-interface" % "0.11" % "test",
+  "com.storm-enroute" %% "scalameter" % "0.8.2"
```

Comment:
BTW, there is another team which is responsible of writing a more polished test suite. In the meanwhile, I think this one can give us a correct estimation of the differences between the various collection implementations.